guide/TEMPLATE.md

```diff
@@ -52,6 +52,14 @@ If you need to update the downloaded repository, just add the flag `--overwrite`
 ./faas-cli template pull https://github.com/itscaro/openfaas-template-php.git --override
 ```
 
+You can specify the template URL with `OPENFAAS_TEMPLATES_URL` environmental variable. CLI overrides the environmental variable.
+
+```bash
+export OPENFAAS_TEMPLATES_URL="https://github.com/itscaro/openfaas-template-php.git"
```

Comment:
Do we have an alternative/project template url we could use here?  I'm thinking about maintainability in the case where the user tries this example and the repo has gone, or it is there and it hasn't been maintained in line with the project.

I'm sure we've had a similar case in a previous PR but I cant remember what we did nor can I find it

Reply:
make sense.